lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoilDX.rb

```diff
@@ -0,0 +1,68 @@
+# A variety of DX coil methods that are the same regardless of coil type.
+# These methods are available to:
+# CoilCoolingDXSingleSpeed, CoilCoolingDXTwoSpeed, CoilCoolingDXMultiSpeed
+module ASHRAEPRMCoilDX
+  # @!group CoilDX
+
+  # Finds the subcategory.  Possible choices are:
+  # Single Package, Split System, PTAC, or PTHP
+  #
+  # @return [String] the coil_dx_subcategory(coil_dx)
+  # @todo Add add split system vs single package to model object
+  def coil_dx_subcategory(coil_dx, capacity, sys_type)
```

Comment:
The methods in this file are missing YARD comments for the parameters.

Reply:
Added